src/App.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import UserForm from "./components/UserForm/UserForm";
+import UserForm from "./components/UserForm/userForm";
```

Comment:
Need to merge with the latest dev branch. This component name has changed from `userForm` to `UserForm`

Reply:
i have merged the latest dev branch, i try to run it after the merge but there is an error pointing to this line. i believe the name here spelt wrong. thats why i have changed this back so as to make the testing functional

Reply:
This is an issue on everybody's branches. We think the problem has occurred because git doesn't recognise a file name changing the case of one letter (i.e. userForm to UserForm) as a change, therefore doesn't change the file name on Github. We will have fix this separately